grpc/server/protobuf/casper/state.proto

```diff
@@ -269,11 +288,25 @@ message BigInt {
 	uint32 bit_width = 2;
 }
 
+message DeployHash {
+    bytes deploy_hash = 1;
+}
+
+message AccountHash {
+    bytes account_hash = 1;
+}
+
+message TransferAddr {
+    bytes transfer_addr = 1;
+}
+
 message Key {
 	oneof value {
 		Address address = 1;
 		Hash hash = 2;
 		URef uref = 3;
+    TransferAddr transfer = 4;
+		DeployHash deploy_info = 5;
```

Comment:
Looks like we have tabs in this file! :scream:

Reply:
Crikey.  Will fix.

Reply:
Fixed.